core/server/worker/src/main/java/alluxio/worker/grpc/AbstractWriteHandler.java

```diff
@@ -59,6 +61,8 @@
 @NotThreadSafe
 abstract class AbstractWriteHandler<T extends WriteRequestContext<?>> {
   private static final Logger LOG = LoggerFactory.getLogger(AbstractWriteHandler.class);
+  private static final Logger SLOW_WRITE_LOG = new SamplingLogger(LOG, 5 * Constants.MINUTE_MS);
+  private static final long SLOW_WRITE_MS = 10 * Constants.SECOND_MS;
```

Comment:
Is there any value in making this configurable?

Reply:
Yeah, I was torn about this. My concern was that there could be too many logging thresholds to configure, which could be confusing.

On the other hand, the hardcoded threshold may not be correct or reasonable.

I guess we could combine the read and write threshold into something like "worker.remote.read.slow.io.threshold"? That could handle both the read and write case?

What do you think?

Reply:
I added a config parameter.

Reply:
Yeah, I think one threshold param would be sufficient.